### PR TITLE
Update error message when user database has lost quorum

### PR DESCRIPTION
**Pull request by jackwaudby**
Update error message encountered on disaster recovery path when recovering servers.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1136.surge.sh

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 1]
```diff
@@ -121,7 +121,7 @@ If *all* servers show health `AVAILABLE` and status `ENABLED` continue to xref:c
 +
 or
 +
-`Could not deallocate server [server]. Database [database] has lost quorum of servers, only found [existing number of primaries] of [expected number of primaries]. Cannot be safely deallocated. Please drop the database before retrying.`
+`Could not deallocate server(s) [server]. Database [database] has lost quorum of servers, only found [existing number of primaries] of [expected number of primaries]. Cannot be safely reallocated.`
```
Comment (AlexicaWright) [inline modules/ROOT/pages/clustering/disaster-recovery.adoc:124]:
```suggestion
`Could not deallocate server(s) [server]. Database [database] has lost quorum of servers, only found [existing number of primaries] of [expected number of primaries]. Cannot be safely reallocated.`
```
```suggestion
`Could not deallocate server(s) [server]. 
Database [database] has lost quorum of servers, only found [existing number of primaries] of [expected number of primaries]. 
Cannot be safely reallocated.`
```

**Review by AlexicaWright — COMMENTED** [review]
One sentence per line. That's all. Thanks @jackwaudby !

**Comment by jackwaudby** [comment]
@AlexicaWright Noted! Thanks!! 🙂

**Pull request merged**